azure-go-sdk-conceptual/azure-sdk-go-authorization.md

```diff
@@ -45,13 +48,13 @@ All authentication functions and types are available in the `github.com/Azure/go
 
 If you're running your application in a tightly controlled environment such as in a container, environment-based authentication is a natural choice. You configure the shell environment before running your application and the Go SDK reads these environment variables at runtime to authenticate with Azure. 
 
-Environment-based authentication has support for all authentication methods except device tokens, evaluated in the following order: Service principal credentials, certificates, username/password, and Managed Service Identity (MSI). If a required environment variable is unset or the SDK gets a refusal from the authentication service, the next authentication type is tried. If the SDK cannot authenticate from the environment, it returns an error.
+Environment-based authentication has support for all authentication methods except device tokens, evaluated in the following order: Client credentials, certificates, username/password, and Managed Service Identity (MSI). If a required environment variable is unset or the SDK gets a refusal from the authentication service, the next authentication type is tried. If the SDK cannot authenticate from the environment, it returns an error.
 
 The following table details the environment variables that need to be set for each authentication type supported by environment-based authentication.
 
 | Authentication type | Environment variable | Description |
 | ------------------- | -------------------- | ----------- |
-| __Service principal__ | `AZURE_TENANT_ID` | The ID for the Active Directory tenant that the service principal belongs to. |
+| __Client credentials__ | `AZURE_TENANT_ID` | The ID for the Active Directory tenant that the service principal belongs to. |
 | | `AZURE_CLIENT_ID` | The name or ID of the service principal. |

```

Comment:
this should be for the AAD app that the SP belongs to

Reply:
I've used the AAD tenant here, rather than AAD app ID. It may be the case that both work. It depends on whether the service principal is used as a single-login configuration for the application or not.